rules/windows/process_creation/proc_creation_win_lolbin_mftrace.yml

```diff
@@ -0,0 +1,28 @@
+title: Use of Mftrace.exe
+id: 3d48c9d3-1aa6-418d-98d3-8fd3c01a564e
+status: experimental
+description: The "Trace log generation tool for Media Foundation Tools" (Mftrace.exe) can be used to execute arbitrary binaries
+author: Nasreddine Bencherchali
+references:
+    - https://lolbas-project.github.io/lolbas/OtherMSBinaries/Mftrace/
+date: 2022/06/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        - Image|endswith: '\mftrace.exe'
+        - OriginalFileName: 'mftrace.exe'
+    selection_cli:
+        # This assumes that when someone use mftrace to launch an ".exe" he will add a space and the binary launched will end with ".exe"
+        CommandLine|startswith: '.exe '
+        CommandLine|endswith: '.exe'
```

Comment:
```yaml
CommandLine|endswith: 
 - '.exe'
 - '.exe '
```
CommandLine|startswith: '.exe '  ?

Reply:
If i only did the "endswith" version this will catch the launch without arguments which is covered by the parent selection. Basically the "startswith" is simply saying that there is a space after the name of the binary.

For example: "mftrace.exe calc.exe" 

I hope i made some sense :)

Reply:
startswith can not exist because CommandLine have to be like ".exe name_of_the_file.exe".
I not sure we can make a '.exe' and lauch it.
So I change `startswith` to `contains`

Reply:
Yes, i meant contains. My brain is not working
Sorry man